### DISCUSSION TITLE
Kestra - automation lowcode plateform

### DISCUSSION THREAD

**Discussion by romainds-tech** [coollabsio/coolify#3944] [Service Template Requests]
it's a good plateform very useful

Official repo https://github.com/kestra-io/kestra

Docker compose installation tutorial https://kestra.io/docs/installation/docker-compose

Docker compose example in repo https://github.com/kestra-io/kestra/blob/develop/docker-compose.yml

**Comment by prithviz** [comment]
PR is ready for review 

https://github.com/coollabsio/coolify/pull/4698

**Reply by romainds-tech** [reply]
Why was the PR closed?
